Makefile.am

```diff
@@ -813,6 +813,18 @@ endif
 
 check: $(CHECK_DEPS)
 
+install-all: install-kernel |  install-image |  install
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	$(srcdir)/installation/install_clear_image.sh $(CLEAR_VM_IMAGE_VERSION) $(IMAGE_PATH)
```

Comment:
This is still potentially going to run `curl` as root if the user is running `sudo make`.

Since the scripts call `sudo` when needed (good), we should be able to stop `install-kernel` and `install-image` running as `root` with something like:

```
ifeq ($(shell id -u), 0)
running_as_root=1
endif

artifacts: | install-image install-kernel

install-kernel:
ifneq ($(running_as_root),)
	$(error ERROR: do not run $@ as root)
endif
	# ...

install-image:
ifneq ($(running_as_root),)
	$(error ERROR: do not run $@ as root)
endif
	# ...
```

This does mean that `sudo make install` won't do everything - we'll have to add a new rule for downloading and installing the artifacts, but the user will then run:

```
make artifacts && sudo make install
```

... which doesn't seem to arduous. We'll need to document how to do this of course.